update debian12 anssi bp28 minimal profile

a-skr opened the pull request:
#### Description:

-  Add some rules that were previously deactivated because they had a prodtype incompatible with the debian12 product
- This work applies to debian12 anssi bp 28 profile.

Mab879 commented:
/packit build

jan-cerny commented:
/packit retest-failed

marcusburghardt commented on shared/templates/accounts_password/ansible.template:
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle,multi_platform_debian
```
```suggestion
# platform = multi_platform_all
```

marcusburghardt commented on shared/templates/accounts_password/bash.template:
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle,multi_platform_debian
```
```suggestion
# platform = multi_platform_all
```

marcusburghardt commented on shared/templates/audit_rules_privileged_commands/bash.template:
```diff
@@ -1,7 +1,7 @@
   {{%- set perm_x=" -F perm=x" %}}
 {{%- endif %}}
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu,multi_platform_debian
```
```suggestion
# platform = multi_platform_all
```

marcusburghardt commented on shared/templates/audit_rules_usergroup_modification/bash.template:
```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu,multi_platform_debian
```
```suggestion
# platform = multi_platform_all
```

a-skr replied:
I guess multi_platform_all will include openembedded and uos20 products too. Is that ok?

marcusburghardt replied:
Yes it is ok. This rule is not used in the only profile present for these products.

a-skr replied:
I will update the commit, then.

marcusburghardt reviewed (changes requested):
The platform value can be simplified in some rules. I also saw that some OVAL and Remediation files were duplicated from Ubuntu. It is not wrong how it was done and probably doesn't fit in the scope of this PR, but we should think in a way reduce duplication in situations like this. @dodys , could you take a look on this PR too, please?

dodys commented:
> The platform value can be simplified in some rules. I also saw that some OVAL and Remediation files were duplicated from Ubuntu. It is not wrong how it was done and probably doesn't fit in the scope of this PR, but we should think in a way reduce duplication in situations like this. @dodys , could you take a look on this PR too, please?

I believe for all those we could create/use a template instead to avoid having separate files. 
Would you like this to be addressed in this PR?

marcusburghardt commented:
> > The platform value can be simplified in some rules. I also saw that some OVAL and Remediation files were duplicated from Ubuntu. It is not wrong how it was done and probably doesn't fit in the scope of this PR, but we should think in a way reduce duplication in situations like this. @dodys , could you take a look on this PR too, please?
> 
> I believe for all those we could create/use a template instead to avoid having separate files. Would you like this to be addressed in this PR?

@a-skr , did you have an opportunity to see this comment from @dodys ?

a-skr commented:
@marcusburghardt  :  I've seen @dodys  comment, but as the comment was interrogative, I wasn't sure something more was expected in this PR. I will look into templates this week.

a-skr commented:
> @marcusburghardt : I've seen @dodys comment, but as the comment was interrogative, I wasn't sure something more was expected in this PR. I will look into templates this week.

By the way the "duplication" of ubuntu oval tests are actually symlinks, so there is no code duplication.

a-skr commented:
@marcusburghardt , @dodys : could you please review the last commit (e1399e2592ee2)? As it is my first template, I would like a review before adding more templates to get rid of the oval symlinks and remaining code duplication.

dodys commented:
> @marcusburghardt , @dodys : could you please review the last commit? As it is my first template, I would like a review before adding more templates to get rid of the oval symlinks and remaining code duplication.

@a-skr, I believe the direction is to always have templates under `shared/templates/`
a minor comment on the templates is just to keep the space indentation as it was. Your editor might be changing it automatically.

Thanks for working on it on this PR :)

a-skr commented:
> > @marcusburghardt , @dodys : could you please review the last commit? As it is my first template, I would like a review before adding more templates to get rid of the oval symlinks and remaining code duplication.
> 
> @a-skr, I believe the direction is to always have templates under `shared/templates/`, just like you did for the tests a minor comment on the templates is just to keep the space indentation as it was. Your editor might be changing it automatically.
> 
> Thanks for working on it on this PR :)

@dodys : can you be more specific with the indentation issue? xml and yaml are indented with two spaces, which seems similar to original files. I think I'm missing something.

dodys commented on shared/templates/pam_account_password_unix_remember/oval.template:
```diff
@@ -1,57 +1,59 @@
+            declaration is common in pam files. -->
         <criterion test_ref="test_accounts_password_pam_unix_remember"
-          comment="pam_pwhistory.so is properly defined in password section of PAM file"/>
+		   comment="pam_pwhistory.so is properly defined in password section of PAM file"/>
```
@a-skr like here in this file
There's a lot of indentation changes.

a-skr replied:
I see...

I mixed content from both ubuntu and shared oval files, but they don't use the same indentation. I will fix it with next PR update.

a-skr commented:
@dodys, @marcusburghardt : I think the PR is now in good shape to be merged.

A pam_account_password_faillock template has been added. So far, only the rules used by Debian use the new template.

If you don't mind, I prefer to refactor the remaining faillock rules in another PR.

marcusburghardt commented:
Could you resolve the conflicts, please?

a-skr commented:
> Could you resolve the conflicts, please?

I will do it this week end.

a-skr commented:
@marcusburghardt I just rebased on master, commit 44b81ca8e3085b8452d8. Looks like there isn't any conflict anymore (no fix needed).

marcusburghardt commented:
/packit build

a-skr commented:
note: commits amended today to solve a merge conflict.

dodys commented:
@a-skr some tests you touched are failing on some platforms. Do you know if that was the case already?

a-skr commented:
> @a-skr some tests you touched are failing on some platforms. Do you know if that was the case already?

Some platform (fedora, ...) were already failing before this PR.
I'm not sure wether this PR add more errors (I did keep the old tests).

Is there is a test suite history that enable me to look at tests on commit bd9ef20a1050268a30856afa5 ?
(That's the one I rebased on).

If so, we will now if I introduced issues.

dodys commented:
> > @a-skr some tests you touched are failing on some platforms. Do you know if that was the case already?
> 
> Some platform (fedora, ...) were already failing before this PR. I'm not sure wether this PR add more errors (I did keep the old tests).
> 
> Is there is a way to run the test suite on commit [bd9ef20](https://github.com/ComplianceAsCode/content/commit/bd9ef20a1050268a30856afa5de74f5d372c29d3) ? (That's the one I rebased on).
> 
> If so, we will now if I introduced issues.

@marcusburghardt is it something you can help?

marcusburghardt reviewed (dismissed):
The CI tests failing for pam_faillock related rules seem legit. I have tested the rules with the master and with the changes in this PR. Only after these changes these rules are failing. These PAM related rules are complex and some rules differ a little bit based on the specific pam_faillock configuration. We should investigate if the issue is with the assessment or with the test scenarios.

a-skr commented on linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/ansible/shared.yml:
```diff
@@ -1,21 +1,69 @@
+{{% set module='pam_unix.so' %}}
+{{% set option='remember' %}}
+{{% set value='{{ var_password_pam_unix_remember }}' %}}
+{{% elif product in [ "sle12", "sle13" ] %}}
```
Thank you.

It is effectively a typo.
I will fix it ASAP.

a-skr commented:
> The CI tests failing for pam_faillock related rules seem legit. I have tested the rules with the master and with the changes in this PR. Only after these changes these rules are failing. These PAM related rules are complex and some rules differ a little bit based on the specific pam_faillock configuration. We should investigate if the issue is with the assessment or with the test scenarios.

I did keep the  old tests unmodified. I found some bugs in the new OVAL faillock template  (commit has been amended).

I also found something that may be an issue with the tests:

I focus on fedora to test and debug. I have this error on the test suite report:

```
ERROR - Script conflicting_settings_authselect.fail.sh using profile (all) found issue:
ERROR - Rule evaluation resulted in pass, instead of expected fail during initial stage 
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny'.
```

But according to platform definitions in the test, this script should not be used with fedora?

```bash
#!/bin/bash
# packages = authselect,pam
# platform = Oracle Linux 8,Oracle Linux 9,Red Hat Enterprise Linux 8,Red Hat Enterprise Linux 9

pam_files=("password-auth" "system-auth")

authselect create-profile testingProfile --base-on minimal
# --- snip ---
```

(by the way, `minimal` is not a profile shipped with fedora, so the script can only fail).

Is the platform definition valid?

jan-cerny commented:
/packit build

jan-cerny commented:
The CI job "Automatus Fedora" runs tests using RHEL 8 content on a Fedora container, the test suite does some special modification to make the tests executable. That usually works but not always. Sometimes, there are problems caused by different nature of containers - for example, there is no running dbus. Sometimes, the problem is that Fedora is different than RHEL 8. In these situations, it is necessary to run the tests locally using a virtual machine back end to determine if the fail is caused by the fedora container environment or by the actual contents of the PR.

I have executed some of the tests on a RHEL 9 virtual machine backed using content built from this PR and they all pass.

```
jcerny@fedora content]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 accounts_password_pam_retry
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/content/logs/rule-custom-2024-05-20-2119/test_suite.log
WARNING - Script correct_value.pass.sh is not applicable on given platform
WARNING - Script wrong_value.fail.sh is not applicable on given platform
INFO - xccdf_org.ssgproject.content_rule_accounts_password_pam_retry
INFO - Script argument_missing.fail.sh using profile (all) OK
INFO - Script pwquality_conf_commented.fail.sh using profile (all) OK
INFO - Script pwquality_conf_conflicting_values.fail.sh using profile (all) OK
INFO - Script pwquality_conf_correct.pass.sh using profile (all) OK
INFO - Script pwquality_conf_correct_with_space.pass.sh using profile (all) OK
INFO - Script pwquality_conf_duplicate_values.pass.sh using profile (all) OK
INFO - Script pwquality_conf_overriden.fail.sh using profile (all) OK
INFO - Script pwquality_conf_wrong.fail.sh using profile (all) OK
[jcerny@fedora content]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 accounts_passwords_pam_faillock_dir
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/content/logs/rule-custom-2024-05-20-2126/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_dir
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script expected_pam_files.pass.sh using profile (all) OK
INFO - Script missing_dir_in_authfail.fail.sh using profile (all) OK
INFO - Script missing_dir_in_preauth.fail.sh using profile (all) OK
INFO - Script wrong_faillock_conf.fail.sh using profile (all) OK
INFO - Script wrong_pam_files.fail.sh using profile (all) OK
[jcerny@fedora content]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel9 accounts_passwords_pam_faillock_deny
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/content/logs/rule-custom-2024-05-20-2130/test_suite.log
WARNING - Script pam_faillock_expected_pam_files.pass.sh is not applicable on given platform
WARNING - Script pam_faillock_lenient_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_multiple_pam_unix_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_stricter_pam_files.pass.sh is not applicable on given platform
WARNING - Script ubuntu_commented_values.fail.sh is not applicable on given platform
WARNING - Script ubuntu_correct.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct_pamd.pass.sh is not applicable on given platform
WARNING - Script ubuntu_empty_faillock_conf.fail.sh is not applicable on given platform
WARNING - Script ubuntu_missing_pamd.fail.sh is not applicable on given platform
WARNING - Script ubuntu_multiple_pam_unix.fail.sh is not applicable on given platform
WARNING - Script ubuntu_wrong_value.fail.sh is not applicable on given platform
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script pam_faillock_conflicting_settings.fail.sh using profile (all) OK
INFO - Script pam_faillock_disabled.fail.sh using profile (all) OK
INFO - Script pam_faillock_expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_lenient_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_multiple_pam_unix_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_not_required_pam_files.fail.sh using profile (all) OK
INFO - Script pam_faillock_stricter_faillock_conf.pass.sh using profile (all) OK
```

Also in the logs from the GitHub job I can see this which I can't see locally

```
+ authselect create-profile testingProfile --base-on minimal
[error] Unable to read base profile [minimal] [2]: No such file or directory
[error] Unable to create profile [2]: No such file or directory
Unable to create new profile [2]: No such file or directory
++ for file in ${configuration_files[@]}
++ sed -i --follow-symlinks '/pam_pwquality\.so/d' /etc/authselect/custom/testingProfile/password-auth
sed: couldn't readlink /etc/authselect/custom/testingProfile/password-auth: No such file or directory
++ for file in ${configuration_files[@]}
```

So I assume the fails aren't caused by this PR but are caused by the environment in the CI job.

jan-cerny reviewed (approved):
I also tried with RHEL 8 

jcerny@fedora:~/work/git/scap-security-guide (pr/11592)$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel8 accounts_passwords_pam_faillock_deny,accounts_password_pam_retry,package_ypbind_removed,package_pam_pwquality_installed,accounts_passwords_pam_faillock_dir,accounts_password_pam_unix_remember,accounts_passwords_pam_faillock_interval,accounts_passwords_pam_faillock_unlock_time,accounts_password_pam_unix_rounds_password_auth
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2024-05-21-1426/test_suite.log
WARNING - Script argument_missing.fail.sh is not applicable on given platform
WARNING - Script correct_value.pass.sh is not applicable on given platform
WARNING - Script wrong_value.fail.sh is not applicable on given platform
WARNING - Script ubuntu_arg_missing.fail.sh is not applicable on given platform
WARNING - Script ubuntu_correct_value.pass.sh is not applicable on given platform
WARNING - Script ubuntu_wrong_value.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_expected_pam_files.pass.sh is not applicable on given platform
WARNING - Script pam_faillock_lenient_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_multiple_pam_unix_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_stricter_pam_files.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct_pamd.pass.sh is not applicable on given platform
WARNING - Script ubuntu_empty_faillock_conf.fail.sh is not applicable on given platform
WARNING - Script ubuntu_multiple_pam_unix.fail.sh is not applicable on given platform
WARNING - Script ubuntu_wrong_value.fail.sh is not applicable on given platform
WARNING - Script ubuntu_commented_values.fail.sh is not applicable on given platform
WARNING - Script ubuntu_missing_pamd.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_expected_pam_files.pass.sh is not applicable on given platform
WARNING - Script pam_faillock_lenient_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_multiple_pam_unix_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_stricter_pam_files.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct_pamd.pass.sh is not applicable on given platform
WARNING - Script ubuntu_empty_faillock_conf.fail.sh is not applicable on given platform
WARNING - Script ubuntu_multiple_pam_unix.fail.sh is not applicable on given platform
WARNING - Script ubuntu_wrong_value.fail.sh is not applicable on given platform
WARNING - Script ubuntu_commented_values.fail.sh is not applicable on given platform
WARNING - Script ubuntu_missing_pamd.fail.sh is not applicable on given platform
WARNING - Script ubuntu_wrong_value.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_expected_pam_files.pass.sh is not applicable on given platform
WARNING - Script pam_faillock_lenient_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_multiple_pam_unix_pam_files.fail.sh is not applicable on given platform
WARNING - Script pam_faillock_stricter_pam_files.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct.pass.sh is not applicable on given platform
WARNING - Script ubuntu_correct_pamd.pass.sh is not applicable on given platform
WARNING - Script ubuntu_empty_faillock_conf.fail.sh is not applicable on given platform
WARNING - Script ubuntu_multiple_pam_unix.fail.sh is not applicable on given platform
WARNING - Script ubuntu_commented_values.fail.sh is not applicable on given platform
WARNING - Script ubuntu_missing_pamd.fail.sh is not applicable on given platform
WARNING - Script correct_value.pass.sh is not applicable on given platform
WARNING - Script wrong_value.fail.sh is not applicable on given platform
WARNING - Script explicit_rounds.pass.sh is not applicable on given platform
WARNING - Script less_rounds.fail.sh is not applicable on given platform
WARNING - Script wrong_control.fail.sh is not applicable on given platform
WARNING - Script default_rounds.fail.sh is not applicable on given platform
INFO - xccdf_org.ssgproject.content_rule_package_ypbind_removed
INFO - Script package-installed.fail.sh using profile (all) OK
INFO - Script package-installed-removed.pass.sh using profile (all) OK
INFO - Script package-removed.pass.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_remember
INFO - Script authselect_conflict_settings.fail.sh using profile (all) OK
INFO - Script authselect_correct_value_conf.pass.sh using profile (all) OK
INFO - Script authselect_correct_value_pam.pass.sh using profile (all) OK
INFO - Script authselect_missing_argument.fail.sh using profile (all) OK
INFO - Script authselect_missing_line.fail.sh using profile (all) OK
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script authselect_pam_unix_legacy_correct.pass.sh using profile (all) OK
INFO - Script authselect_pam_unix_legacy_wrong.fail.sh using profile (all) OK
INFO - Script authselect_wrong_value_conf.fail.sh using profile (all) OK
INFO - Script authselect_wrong_value_pam.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_deny
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script pam_faillock_conflicting_settings.fail.sh using profile (all) OK
INFO - Script pam_faillock_expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_lenient_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_multiple_pam_unix_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_stricter_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_disabled.fail.sh using profile (all) OK
INFO - Script pam_faillock_not_required_pam_files.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_dir
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script expected_pam_files.pass.sh using profile (all) OK
INFO - Script missing_dir_in_authfail.fail.sh using profile (all) OK
INFO - Script missing_dir_in_preauth.fail.sh using profile (all) OK
INFO - Script wrong_faillock_conf.fail.sh using profile (all) OK
INFO - Script wrong_pam_files.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_interval
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script pam_faillock_conflicting_settings.fail.sh using profile (all) OK
INFO - Script pam_faillock_expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_lenient_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_multiple_pam_unix_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_stricter_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_disabled.fail.sh using profile (all) OK
INFO - Script pam_faillock_not_required_pam_files.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_passwords_pam_faillock_unlock_time
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script conflicting_settings_authselect.fail.sh using profile (all) OK
INFO - Script pam_faillock_conflicting_settings.fail.sh using profile (all) OK
INFO - Script pam_faillock_expected_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_lenient_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_multiple_pam_unix_faillock_conf.fail.sh using profile (all) OK
INFO - Script pam_faillock_stricter_faillock_conf.pass.sh using profile (all) OK
INFO - Script pam_faillock_disabled.fail.sh using profile (all) OK
INFO - Script pam_faillock_not_required_pam_files.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_package_pam_pwquality_installed
INFO - Script package-installed.pass.sh using profile (all) OK
INFO - Script package-removed.fail.sh using profile (all) OK
INFO - Script package-installed-removed.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_password_pam_retry
INFO - Script argument_missing.fail.sh using profile (all) OK
INFO - Script pwquality_conf_commented.fail.sh using profile (all) OK
INFO - Script pwquality_conf_conflicting_values.fail.sh using profile (all) OK
INFO - Script pwquality_conf_correct.pass.sh using profile (all) OK
INFO - Script pwquality_conf_correct_with_space.pass.sh using profile (all) OK
INFO - Script pwquality_conf_duplicate_values.pass.sh using profile (all) OK
INFO - Script pwquality_conf_overriden.fail.sh using profile (all) OK
INFO - Script pwquality_conf_wrong.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_accounts_password_pam_unix_rounds_password_auth
INFO - Script authselect_argument_missing.fail.sh using profile (all) OK
INFO - Script authselect_correct_value.pass.sh using profile (all) OK
INFO - Script authselect_modified_pam.fail.sh using profile (all) OK
INFO - Script authselect_wrong_control.fail.sh using profile (all) OK
INFO - Script authselect_wrong_value.fail.sh using profile (all) OK
INFO - Script authselect_default_rounds.fail.sh using profile (all) OK
